### ISSUE TITLE
[BUG] Sometimes YouTube won't let me get into landscape mode while watching a video (by physically rotating a video), and when I click the FullScreen button, it just gets messed up (see comment for video link)

### ISSUE THREAD

**Issue by karaigumon** [eni9889/YT-PP-Issues#31]
iOS Version: 10.2
 YouTube Version: 12.21
 YT ++ Tweak Version: 
 Crash Report: No Crash
 Jailbreak

Issue is mostly explained in the title.  here is the video showing it:

https://youtu.be/bFlcCFzAr9M

**Comment by JMccormick264** [comment]
Does this only happen when using YouTube++?

**Comment by karaigumon** [comment]
yes.  When I remove youtube++ it works fine.  but whenever I have YT++, it only works for like the first 2 videos maybe, and then the glitch mentioned above happens.
